From c3ebc9085cb5288de24f287ba9dc29d876363fe1 Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Mon, 27 Jan 2020 12:04:06 +0100
Subject: [PATCH 1074/1239] net: mvpp2: sort headers

This patch adds an alphabetical sorting of the included
headers. No functional change.

Change-Id: Ic97c14cb7e39e4996e7d1e6aa591da728c93f224
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/22361
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
---
 drivers/net/mvpp2.c | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/drivers/net/mvpp2.c b/drivers/net/mvpp2.c
index 59dfb366bc..41afb7ed22 100644
--- a/drivers/net/mvpp2.c
+++ b/drivers/net/mvpp2.c
@@ -13,26 +13,26 @@
  * warranty of any kind, whether express or implied.
  */
 
+#include <asm-generic/gpio.h>
+#include <asm/arch/cpu.h>
+#include <asm/arch/soc.h>
+#include <asm/io.h>
 #include <common.h>
+#include <config.h>
 #include <dm.h>
 #include <dm/device-internal.h>
 #include <dm/lists.h>
+#include <fdt_support.h>
+#include <linux/compat.h>
+#include <linux/errno.h>
+#include <linux/mbus.h>
+#include <linux/mdio.h>
+#include <malloc.h>
+#include <miiphy.h>
 #include <net.h>
 #include <netdev.h>
-#include <config.h>
-#include <malloc.h>
-#include <asm/io.h>
-#include <linux/errno.h>
 #include <phy.h>
-#include <miiphy.h>
 #include <watchdog.h>
-#include <asm/arch/cpu.h>
-#include <asm/arch/soc.h>
-#include <linux/compat.h>
-#include <linux/mbus.h>
-#include <asm-generic/gpio.h>
-#include <fdt_support.h>
-#include <linux/mdio.h>
 
 DECLARE_GLOBAL_DATA_PTR;
 
-- 
2.29.0

